js/main.js

```diff
@@ -0,0 +1,18 @@
+import './user-form.js';
+import './map.js';
+import './filter.js';
+import {createAllMarkers} from './map.js';
+import {showAlert} from './message.js';
+import {getData} from './api.js';
+import {setOnFiltersChange} from './filter-user.js';
+
+
+const ADS_NUM = 10;
```

Comment:
```suggestion
const ADS_LIMIT = 10;
```
Мне кажется так более точно отражается суть.